L1Trigger/Phase2L1Taus/src/lutAuxFunctions.cc

```diff
@@ -0,0 +1,62 @@
+#include "L1Trigger/Phase2L1Taus/interface/lutAuxFunctions.h"
+
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include <TFile.h> // TFile
+#include <TH1.h>   // TH1
+#include <TH2.h>   // TH2
+
+/**
+ * @brief Open ROOT file the name of which is given as function argument
+ * @param fileName: name of ROOT file
+ * @return pointer to TFile object
+ */
+TFile *
+openFile(const LocalFileInPath & fileName)
+{
+  if(fileName.fullPath().empty())
+  {
+    throw cms::Exception("openFile") 
+      << " Failed to find file = " << fileName << " !!\n";
+  }
+  TFile * inputFile = new TFile(fileName.fullPath().data());
```

Comment:
@sandeepbhowmik1 
Please make a follow-up PR with replacing this with a smart pointer.